live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -175,3 +175,18 @@
         default_user:
           name: {{ lookup('env', 'APPLIANCE_USERNAME') }}
           lock_passwd: false
+
+#
+# IPv6 addresses are not supported by the product thus we need
+# to ensure that they cannot be added manually by default. In
+# addition, we wouldn't want to have our secondary addresses
+# deleted when someone deletes the primary IP address of a
+# network interface, thus we enable the promotion of secondary
+# addresses to primary ones.
+#
+- blockinfile:
```

Comment:
I'm not against using `blockinfile`, but I'm curious why you opted to use this rather than `lineinfile`?

E.g. I think we could accomplish this like so:
```
- lineinfile:
    dest: /etc/sysctl.conf
    regexp: "^#?{{ item.key }}="
    line: "{{ item.key }}={{ item.value }}"
  with_items:
    - { key: "net.ipv4.conf.all.promote_secondaries", value: "1" }
    - { key: "net.ipv6.conf.all.disable_ipv6", value: "1" } 
```

I wonder.. if these either of these two keys are already present in `/etc/sysctl.conf`, will the prior values get overwritten when using `blockinfile`? or will the old line still exist, and we'll simply add another line for the same key? In my example, with `lineinfile`, I think any value already existing in the file would get replaced with the line/value that we want.

Is having two lines for the same key a problem? Do we expect for these values to already be set in this file?